### PR TITLE
Misty/project page polish

### PR DESCRIPTION
**Pull request by mistyb01**
## What

- Updated the Project's page content for when we release the site publicly.
- Minor fix in padding for the project card component.
- Renamed `project_individual` to `project_cadre` (also still need to component-ize the individual project page!)

## Show Me

image

### CODE REVIEW

File: components/cards/CardProject.tsx [hunk 1]
```diff
@@ -30,10 +30,15 @@ const CardProject = ({
...
+    <Card
+      sx={{
+        boxShadow:
+          '0px 2px 4px 0px rgba(52, 61, 62, 0.04), 0px 4px 8px 2px rgba(52, 61, 62, 0.04)',
+      }}
+    >
       <CardContent
         sx={{
-          padding: '2rem',
+          padding: { xs: '2rem 1rem', lg: '2rem' },
```
Comment (santina) [inline components/cards/CardProject.tsx:41]: is this the fix for this comment?

Comment (santina) [inline components/cards/CardProject.tsx:41]: is this the fix for this comment? 

image

**Review by santina — APPROVED** [review]
Thanks! Hopefully we start putting in projects soon!

File: components/cards/CardProject.tsx (hunk 1 above)
Comment (mistyb01) [reply components/cards/CardProject.tsx:41]: not necessarily, but I did add another commit that does set a maxWidth on the page content. thanks for pointing it out!

**Pull request merged**